man/man8/zpool.8

```diff
@@ -2045,6 +2063,17 @@ Because scrubbing and resilvering are \fBI/O\fR-intensive operations, \fBZFS\fR
 .RS 6n
 Stop scrubbing.
 .RE
+.sp
+.ne 2
+.na
+\fB\fB-p\fR\fR
+.ad
+.RS 6n
+Pause scrubbing. Scrub progress is periodically synced to disk so if the system
+is restarted or pool is exported during a paused scrub, the scrub will resume
+from the place where it was last checkpointed to disk. To resume a paused scrub
+issue \fBzpool scrub\fR again.
```

Comment:
I don't see where the paused-ness is persisted to disk.  So when the pool is opened/imported, it will not be paused, even if it was paused before export/reboot (unless you do `zpool import -p`)?  We should at least document that confusing behavior.  But I would much prefer to fix it before integrating this.

I think you could either:
1. add a feature flag and introduce a new dsl_scan_state_t; or 
2. add a new dsl_scan_flags_t, which would be ignored on older software (so if the pool is imported/opened with older software, the scrub would not be paused)